contracts/index_executor/src/lib.rs

```diff
@@ -296,8 +281,59 @@ mod index_executor {
             Ok(())
         }
 
+        #[ink(message)]
+        pub fn pause_executor(&mut self) -> Result<()> {
```

Comment:
when should we pause an executor?

Reply:
When we don't want to engine run task, for instance, we plan to do `approve` to a new contract, we don't want to worker execute task at the same time bc it may lead issue on `nonce`, we may use `nonce` that already allocated to a task